stream_alert_cli.py

```diff
@@ -117,6 +117,13 @@ def build_parser():
         help='Enable DEBUG logger output'
     )
 
+    lambda_parser.add_argument(
+        '-r', '--rules',
+        type=str,
```

Comment:
I knew there had to be an easier way to do that. using `nargs='+'` since at least one should always be provided if this switch is used.